centreon/doc/API/latest/onPremise/Realtime/Resources/Schema/ResourceServiceDetail.yaml

```
@@ -1,438 +1,441 @@
-              type: string
-              nullable: true
-              description: "Service resource status graph endpoint"
-              example: "/centreon/api/v22.04/monitoring/hosts/14/services/26/metrics/status"
-            performance_graph:
-              type: string
-              nullable: true
-              description: "Service resource performance graph endpoint"
-              example: "/centreon/api/v22.04/monitoring/hosts/14/services/26/metrics/performance"
\ No newline at end of file
+            description: "Service resource performance graph endpoint"
+            example: "/centreon/api/v22.04/monitoring/hosts/14/services/26/metrics/performance"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)